tests/hazmat/primitives/test_rsa.py

```diff
@@ -1981,10 +1981,10 @@ def test_recover_prime_factors(self, vector):
             private["private_exponent"]
         )
         # Unfortunately there is no convention on which prime should be p
-        # and which one q. The function we use always makes p < q, but the
+        # and which one q. The function we use always makes p > q, but the
         # NIST vectors are not so consistent. Accordingly, we verify we've
         # recovered the proper (p, q) by sorting them and asserting on that.
-        assert sorted([p, q]) == sorted([private["p"], private["q"]])
+        assert sorted([q, p]) == sorted([private["p"], private["q"]])
```

Comment:
This line should probably be `assert [p, q] == sorted([private["p"], private["q"]], reverse=True)` now. Reordering `[q, p]` inside `sorted` won't actually work since it'll sort by smallest first.